nameable_decorator/decorators.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super()
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super()
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
+  end
+end
```

Comment:
[OPTIONAL] May I know the purpose of using `super()` in `correct_name` method? In my opinion, the `super` is not necessary since you redefined the method in TrimmerDecorator class.

Reply:
Thanks